### PR TITLE
Clean up page indexing

### PR DESCRIPTION
**Pull request by ConnorCimowsky**
Now that display indices are updated in the backend, we can get rid of the old client code.

Note: reordering is still broken. this is just for adding new pages

### CODE REVIEW

**Comment by Kapin** [comment]
LGTM

**Comment by Ommy** [comment]
+1

File: src-frontend/app/controllers/edit-page-modal.js [hunk 1]
```diff
@@ -8,27 +8,23 @@ export default Ember.Controller.extend({
...
-                element.incrementProperty('displayIndex');
-                // TODO: Find a way to batch these updates
-                element.save();
-            });
+            var page = this.get('model');
 
             var newElement = this.store.createRecord('element', {
-                page: this.get('model'),
+                page: page,
                 displayIndex: targetIndex
```
Comment (Trinovantes) [inline src-frontend/app/controllers/edit-page-modal.js]: You deleted `targetIndex`. Shouldn't this be `selectedIndex`?

File: src-frontend/tests/unit/helpers/plus-one-test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { plusOne } from '../../../helpers/plus-one';
+import { module, test } from 'qunit';
+
+module('Unit | Helper | plus one');
+
+// Replace this with your real tests.
+test('it works', function(assert) {
+  var result = plusOne(42);
+  assert.ok(result);
```
Comment (Trinovantes) [inline src-frontend/tests/unit/helpers/plus-one-test.js]: shouldn't this test the value of `result`?

File: src-frontend/app/controllers/edit-page-modal.js (hunk 1 above)
Comment (ConnorCimowsky) [inline src-frontend/app/controllers/edit-page-modal.js]: Yup. Haven't really fixed the element stuff yet, but this is definitely wrong. Thanks!

File: src-frontend/tests/unit/helpers/plus-one-test.js (hunk 2 above)
Comment (ConnorCimowsky) [inline src-frontend/tests/unit/helpers/plus-one-test.js]: Eventually. This is just boilerplate from `ember generate`

**Pull request merged**